data/core.yaml

```diff
@@ -345,6 +345,12 @@ en:
       not_downloaded:
         single: This feature can't be moved because parts of it have not yet been downloaded.
         multiple: These features can't be moved because parts of them have not yet been downloaded.
+    follow:
```

Comment:
Lets also include this capability in 'Help' menu.

Reply:
👍 done